File: WhereToPlay/app/src/main/java/com/fanc/wheretoplay/fragment/AboutUsFragment.java
package com.fanc.wheretoplay.fragment;

import android.databinding.DataBindingUtil;
import android.os.Bundle;
import android.support.annotation.Nullable;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

import com.fanc.wheretoplay.R;
import com.fanc.wheretoplay.base.BaseFragment;
import com.fanc.wheretoplay.databinding.FragmentAboutUsBinding;
import com.fanc.wheretoplay.util.UIUtils;
import com.fanc.wheretoplay.view.ItemView;
import com.fanc.wheretoplay.view.TopMenu;

/**
 * Created by Administrator on 2017/6/19.
 */

public class AboutUsFragment extends BaseFragment {
    FragmentAboutUsBinding aboutUsBinding;

    TopMenu mTmAboutUs;
    ItemView mIvAboutUsVersion;

    @Nullable
    @Override
    public View onCreateView(LayoutInflater inflater, @Nullable ViewGroup container, @Nullable Bundle savedInstanceState) {
        aboutUsBinding = DataBindingUtil.inflate(inflater, R.layout.fragment_about_us, container, false);
        initViews();
        init();
        setListener();
        return aboutUsBinding.getRoot();
    }

    private void initViews() {
        mTmAboutUs = aboutUsBinding.tmAboutUs;
        mIvAboutUsVersion = aboutUsBinding.ivAboutUsVersion;
    }

    private void init() {
        mTmAboutUs.setLeftIcon(R.drawable.left);
        mTmAboutUs.setTitle(R.string.about_us);

        displayCurrentVersion();

    }

    private void setListener() {
        mTmAboutUs.setLeftIconOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                mContext.finish();
            }
        });
    }

    /**
     * 显示当前版本
     */
    private void displayCurrentVersion() {
        mIvAboutUsVersion.setIcon(R.drawable.version);
        mIvAboutUsVersion.setText(R.string.current_version);
        mIvAboutUsVersion.setTextColor(mContext.getResources().getColor(R.color.ddialog_text_color));
        mIvAboutUsVersion.setRightTextColor(mContext.getResources().getColor(R.color.ddialog_text_color));
        mIvAboutUsVersion.setBackgroundColor(mContext.getResources().getColor(R.color.about_us_version));
        mIvAboutUsVersion.isShowLine(false);

        mIvAboutUsVersion.setRightText("v" + UIUtils.getAppVersionName());

    }

}

File: WhereToPlay/app/src/main/java/com/fanc/wheretoplay/fragment/AboutUsFragment.kt
package com.fanc.wheretoplay.fragment

import android.graphics.Color
import android.os.Bundle
import android.support.v4.content.ContextCompat
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup

import com.fanc.wheretoplay.R
import com.fanc.wheretoplay.base.BaseFragment

import com.fanc.wheretoplay.util.UIUtils
import kotlinx.android.synthetic.main.fragment_about_us.*

/**
 *
 * @author Administrator
 * @date 2017/6/19
 */

class AboutUsFragment : BaseFragment() {


    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View? {

        return View.inflate(inflater.context, R.layout.fragment_about_us, null)
    }

    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
        super.onViewCreated(view, savedInstanceState)

        tm_about_us.setLeftIcon(R.drawable.left)
        tm_about_us.setTitle(R.string.about_us)
        tm_about_us.setTitleColor(Color.parseColor("#FFFFFF"))
        tm_about_us.setLeftIconOnClickListener {
            mContext.finish()
        }
        iv_about_us_version.setIcon(R.drawable.version)
        iv_about_us_version.setText(R.string.current_version)
        iv_about_us_version.setTextColor(ContextCompat.getColor(mContext, R.color.ddialog_text_color))
        iv_about_us_version.setRightTextColor(ContextCompat.getColor(mContext, R.color.ddialog_text_color))
        iv_about_us_version.setBackgroundColor(ContextCompat.getColor(mContext, R.color.about_us_version))
        iv_about_us_version.isShowLine(false)

        iv_about_us_version.rightText = "v" + UIUtils.getAppVersionName()
    }


}
